checkstyle/suppressions.xml

```diff
@@ -20,7 +20,7 @@
     <suppress files="(KafkaConsumerGroupClientImpl.java)" checks="ClassDataAbstractionCouplingCheck"/>
     <suppress files="(KsqlResource|KsqlRestApplication|StandaloneExecutor|KsqlRestClient).java" checks="ClassDataAbstractionCouplingCheck"/>
     <suppress files="Console.java" checks="ClassDataAbstractionCouplingCheck"/>
-    <suppress files="(DataGenProducer|DataGen|Generator|PhysicalPlanBuilder).java"
+    <suppress files="(DataGenProducer|DataGen|Generator|PhysicalPlanBuilder|StatementRewriter).java"
```

Comment:
don't add supressions - fix the code.

Reply:
To be fair i think it is a bit difficult for this class to not have the suppression. It is

Reply:
Agree with @dguy . This one needs to instantiate many other classes to build a new AST.

Reply:
Fair enough.